circle.yml

```diff
@@ -9,7 +9,7 @@ jobs:
       XCODE_TEST_REPORTS: /tmp/xcode-test-results
       LANG: en_US.UTF-8
     macos:
-      xcode: '10.1.0'
+      xcode: '10.0.0'
```

Comment:
I don't see a reason to take this down to 10.0 , the failure doens't seem related to the xcode version. I just reran master and it passed with no issues: https://circleci.com/gh/RxSwiftCommunity/Action/85

Reply:
This is really strange.. I will try it again let see if it passes correctly